### PR TITLE
Use page.url in og:image meta data

### PR DESCRIPTION
**Pull request by simonwhatley**

### CODE REVIEW

File: app/posts/apply-for-teacher-training/2022-06-22-home-fee-status.md [hunk 1]
```diff
@@ -17,17 +17,17 @@ related:
...
+- [User need](/apply-for-teacher-training/home-fee-status/#user-need)
+- [Hypothesis](/apply-for-teacher-training/home-fee-status/#hypothesis)
+- [Nationality and residency section](/apply-for-teacher-training/home-fee-status/#nationality-and-residency-section)
+- [British and Irish nationals](/apply-for-teacher-training/home-fee-status/#british-and-irish-nationals)
+- [EU nationals with EU settled status](/apply-for-teacher-training/home-fee-status/#eu-nationals-with-eu-settled-status)
+- [EU nationals with EU pre-settled status](/apply-for-teacher-training/home-fee-status/#eu-nationals-with-eu-pre-settled-status)
+- [Candidates with indefinite leave to remain or right of abode](/apply-for-teacher-training/home-fee-status/#candidates-with-indefinite-leave-to-remain-or-right-of-abode)
+- [Another immigration status](/apply-for-teacher-training/home-fee-status/#another-immigration-status)
+- [Resources](/apply-for-teacher-training/home-fee-status/#resources)
+- [Next steps](/apply-for-teacher-training/home-fee-status/#next-steps)
```
Comment (frankieroberto) [inline app/posts/apply-for-teacher-training/2022-06-22-home-fee-status.md:30]: What's the reason for this change? Thought both bullet styles were fine, and if anything, the asterisks are more common? 🤷

**Review by frankieroberto — APPROVED** [review]
Nice!

File: app/posts/apply-for-teacher-training/2022-06-22-home-fee-status.md (hunk 1 above)
Comment (simonwhatley) [reply app/posts/apply-for-teacher-training/2022-06-22-home-fee-status.md:30]: Most posts seem to use hyphens, so I updated them to match.

It's pedantry more than anything.

Comment (frankieroberto) [reply app/posts/apply-for-teacher-training/2022-06-22-home-fee-status.md:30]: I don't mind pedantry, but if we want consistency on this it might be better to get the existing `markdownlint` to enforce it somehow (maybe there's a rule that could be added)?  There's a GitHub workflow setup which will autofix any issues so once it's setup people don't have to think about it.
